src/perfcollect/perfcollect

```diff
@@ -1171,13 +1180,25 @@ ProcessArguments()
         FatalError "Invalid action specified."
     fi
 
+    # Validate operation name.
+    if [ "$action" == "collect" ]
+    then
```

Comment:
Overall, this seems pretty clean.  I would like to ask for a few changes, which I'll add here in a few comments.  The most structural request is this: Rather than having the command be "collect start" or "collect stop" can you just make it "start" or "stop"?  Then you can leave "collect" as is.  This is what we do in PerfView, so the behavior will extend well to PerfCollect.

Reply:
Having "start/stop" only sounds good to me! Will change.